add new mutation for update Kontoverbindung (#43)
europace/kex-vorgang-update-api: README.md

> Delete an existing Kind. The Haushaltsposition is referenced by the `id`.

### Update Kontoverbindung

**updateKontoverbindung** ( vorgangsnummer: String!, kontoverbindung [Kontoverbindung](#kontoverbindung)! ) -> [BasicResponse](#basicresponse)!

> > Update an existing Kontoverbindung.

Reviewer: ```suggestion
> Update the Kontoverbindung.
```

> #### Hints
> 
> * If `iban` or `bic` are invalid, the value will be ignored and set to `null`.
> 
> **updateKontoverbindung** ( vorgangsnummer: String!, kontoverbindung [Kontoverbindung](#kontoverbindung)! ) -> [BasicResponse](#basicresponse)!
> 
> > Update an existing Kontoverbindung.
> 

Reviewer: ```suggestion
**updateKontoverbindung** ( vorgangsnummer: String!, kontoverbindung [Kontoverbindung](#kontoverbindung)! ) -> [BasicResponse](#basicresponse)!

> Update an existing Kontoverbindung.

#### Hints

* If `iban` or `bic` are invalid, the value will be ignored and set to `null`.

```
Damit es einheitlich ist, sollten die Hints unterhalb der Mutation stehen. Unterhalb der Unterschrift stehen sie nur wenn es allgemeine Sachen sind, die für mehrere Mutations gelten.

> Delete an existing Kind. The Haushaltsposition is referenced by the `id`.

### Update Kontoverbindung

#### Hints

> * The Haushaltsposition `Kontoverbindung` has to be related to at least one Antragsteller.

Reviewer: Den Punkt würde ich eigentlich komplett streichen, das sieht man ja anhand der Mutation das das benötigt wird und die Hinweise sind ja eher Sachen, die man nicht direkt erkennen kann.

Author: @sgrell wenn sich Hinweise nur an Dinge beziehen die man nicht direkt erkennen kann dann stimme ich dir zu.

> * The `antragstellerId` has to refer to an existing Antragsteller in the Vorgang.

Reviewer: ```suggestion
* Every entry in `antragstellerIds` has to correspond to an Antragsteller in the Vorgang.
```
Hier würde ich das gleiche Wording wie bei allen anderen Haushaltspositionen verwenden. Wenn es unter den Haushaltspositionen bleibt, sollte der Punkt allerdings komplett weg, weil es ja die allgemeinen Hinweise gibt.

> * The Haushaltsposition `Kontoverbindung` is only updateable by valid values for the fields `iban` and `bic`.

Reviewer: ```suggestion
* If `iban` or `bic` are invalid, the value will be ignored and set to `null`.
```

Der Satz gefällt mir noch nicht so richtig, aber er trifft es für mich besser als dein Vorschlag, weil man das meiner Meinung nach missverstehen kann, dass dann die alten Werte weiterhin gelten.

        "unterhaltseinnahmenMonatlich": BigDecimal
    }

### Kontoverbindung

    {
        "bic": String,

>         "gehoertZuAntragsteller": Antragstellerzuordnung,

Reviewer: ```suggestion
       "antragstellerIds": [ String ]
```
